client/reader/sidebar/reader-sidebar-recent/index.tsx

```diff
@@ -75,6 +77,17 @@ const ReaderSidebarRecent = ( {
 		if ( ! RECENT_PATH_REGEX.test( path ) ) {
 			page( '/read' );
 		}
+
+		// Analytics.
+		if ( feedId ) {
+			recordAction( 'clicked_reader_sidebar_followed_single_site' );
+			recordGaEvent( 'Clicked Reader Sidebar Followed Single Site' );
+			dispatch( recordReaderTracksEvent( 'calypso_reader_sidebar_followed_single_site_clicked' ) );
```

Comment:
Added new events for selection on a particular feed. Let me know if there is any feedback on it.

Will open the PR for `calypso_reader_sidebar_followed_single_site_clicked` event registration.

Reply:
Yes.